pvlib/solarposition.py

```diff
@@ -27,10 +27,16 @@
 from pvlib.tools import datetime_to_djd, djd_to_datetime
 
 
-def get_solarposition(time, latitude, longitude,
-                      altitude=None, pressure=None,
-                      method='nrel_numpy',
-                      temperature=12, **kwargs):
+def get_solarposition(
+    time,
+    latitude,
+    longitude,
+    altitude,
+    pressure = None,
+    method = 'nrel_numpy',
+    temperature = 12.,
+    **kwargs
+):
```

Comment:
```suggestion
def get_solarposition(time, latitude, longitude,
                      altitude=None, pressure=None,
                      method='nrel_numpy',
                      temperature=12.0, **kwargs):
```

No real need to change the formatting here, so might as well minimize the diff.  I think this will also get the tests working again (as `altitude` is now back to being optional).